addon/initializers/prototype-extend.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+export function initialize() {
+  const { EmberENV } = config;
+
+  if (EmberENV.EXTEND_PROTOTYPES && EmberENV.EXTEND_PROTOTYPES.Function === true && Ember.immediateObserver) {
+    Function.prototype.observesImmediately = Ember.immediateObserver;
```

Comment:
Can we move this to a vendor file as well? Sorry for not noticing until the last minute :(